.github/semantic.yml

```diff
@@ -0,0 +1,13 @@
+# By default types specified in commitizen/conventional-commit-types is used.
+# See: https://github.com/commitizen/conventional-commit-types/blob/v2.3.0/index.json
+# You can override the valid types
+types:
+  - feat
+  - fix
+  - improvement
+  - docs
+  - refactor
+  - test
+  - ci
+  - chore
+  - skip/changelog
```

Comment:
Let's also skip standard prefixes for skipping CI - `[ci skip]`, `[skip ci]`. I am not sure if we have any of those already.

Reply:
I added `skip ci` and `skip changelog` sementics.